pkg/pillar/types/dns.go

```diff
@@ -31,6 +31,8 @@ type DeviceNetworkStatus struct {
 
 type NetworkPortStatus struct {
 	IfName       string
+	USBAddr      string
```

Comment:
I think if you add `USBAddr` then you should also add `USBProd`.

Reply:
@christoph-zededa NetworkPortConfig doesn't currently have a USBProd field.  Is USBAddr enough of an identifier?

Reply:
I also think it makes sense to add that field since we allow specifying only the USBProd for USB devices.

Reply:
F.e. we might use a USB ethernet adapter and specify it in the model with vendor id and product id.

Reply:
Ok i'll make the changes

Reply:
@christoph-zededa @eriknordmark pushed up changes plumbing the USBProd field up through NetworkPortConfig and NetworkPortStatus